docs/release-checklist.md

```diff
@@ -36,7 +36,7 @@
 1. Validate VSCode extension and Bicep CLI manually on Windows, Mac & Linux:
     1. Download `vscode-bicep.vsix` from the draft release, and [Install it from VSIX](https://code.visualstudio.com/docs/editor/extension-marketplace#_install-from-a-vsix). Verify that you can open a Bicep file, that text is correctly colorized, and that error messages show up as expected.
     1. Download the appropriate Bicep executable for your platform (e.g. `bicep-linux-x64`). Verify you can invoke it with e.g. `bicep-linux-x64 --version`, and that it prints the expected output.
-1. Verify that the draft release on GitHub has at least 27 artifacts associated with it before publishing it.
+1. Verify that the draft release on GitHub has at least 27 artifacts associated with it before publishing it. (On the Bicep releases [page](https://github.com/Azure/bicep/releases) it should state 27 assets. On the edit page of the draft release itself, there will be 25 assets because the 2 source code files don't show up.)
```

Comment:
```suggestion
1. Verify that the draft release on GitHub has at least 27 artifacts associated with it before publishing it. (On the Bicep releases [page](https://github.com/Azure/bicep/releases) it should state that many assets. On the edit page of the draft release itself, there will be two fewer assets because the 2 source code files don't show up.)
```

Reply:
future-proofing...